.github/workflows/ci.yml

```diff
@@ -124,22 +126,24 @@ jobs:
     strategy:
       matrix:
         rails_version: [5.2.3, 6.1.1, main]
-        ruby_version: [2.5.x, 2.6.x, 2.7.x, 3.0.x]
+        ruby_version: [2.6.x, 2.7.x, 3.0.x, 3.1.x]
         exclude:
           - rails_version: 5.2.3
             ruby_version: 2.6.x
           - rails_version: 5.2.3
             ruby_version: 2.7.x
           - rails_version: 5.2.3
             ruby_version: 3.0.x
+          - rails_version: 5.2.3
+            ruby_version: 3.1.x
           - rails_version: 6.1.1
-            ruby_version: 2.5.x
+            ruby_version: 2.6.x
           - rails_version: 6.1.1
             ruby_version: 2.7.x
           - rails_version: 6.1.1
             ruby_version: 3.0.x
-          - rails_version: main
-            ruby_version: 2.5.x
+          - rails_version: 6.1.1
+            ruby_version: 3.1.x
           - rails_version: main
             ruby_version: 2.6.x
```

Comment:
Ruby 3.1 plz :)

Reply:
maybe I am tripping but can you check the latest changes?

Reply:
ohhh line 144 against `main`? is there a reason for this being behind so many?

Reply:
done in https://github.com/primer/view_components/pull/1220/commits/c06b8b175e7bee1d87d445d0c28128c689a7053e.

Reply:
Thanks! Yeah I think it was just an oversight.